share/wake/lib/system/remote_cache_api.wake

```diff
@@ -201,7 +201,24 @@ export def makeRemoteCacheApi (config: String): Result RemoteCacheApi Error =
     def auth = if authStr ==* "" then None else Some authStr
     def api = RemoteCacheApi domain port auth
 
-    # TODO: check for compatable wake version
+    # When in debug mode, allow server incompatible version.
+    def overrideOnDebugCache check =
+        require Fail err = check
+        else check
+
+        require Some _ = getenv "DEBUG_WAKE_SHARED_CACHE"
+        else check
+
+        printlnLevel
+        logWarning
+        "RSC client is incompatable with server but continuing due to DEBUG enabled. Reason: {format err}"
+        | Pass
+
+    # If the client isn't compatiable with the server then give up on the cache
+    require Pass Unit =
+        api
+        | rscApiCheckClientVersion "sifive/wake/{version}"
```

Comment:
Where does the variable `version` come from here?

Reply:
its a wake `prim` that returns the same number from `wake --version`

Reply:
(in fact both cases source it from the same spot)